b2/account_info/abstract.py

```diff
@@ -176,8 +182,16 @@ def allowed_is_valid(cls, allowed):
 
     @abstractmethod
     def _set_auth_data(
-        self, account_id, auth_token, api_url, download_url, minimum_part_size, application_key,
-        realm, allowed
+        self,
+        account_id,
+        auth_token,
+        api_url,
+        download_url,
+        minimum_part_size,
+        application_key,
+        realm,
+        allowed,
+        account_id_or_app_key_id,
```

Comment:
So one way to do this would have just been to save the appKeyId in the same area as accountId.
That way you dont have to add a new column and it should never be None. 

The other way is to make a new field.

Any thoughts on if just changing the name of account_id to account_id_or_app_key_id like the new field?  Is there a reason to keep just the accountId with an applicationKeyId?